SLC-S-GQIMonitor/SLC-AS-GQIMonitor.VersionCheck/SLC-AS-GQIMonitor.VersionCheck.cs

```diff
@@ -73,33 +73,71 @@ private void RunSafe(IEngine engine)
 			UIBuilder uiBuilder = new UIBuilder
 			{
 				RequireResponse = true,
-				RowDefs = "a;a",
+				RowDefs = "a;a;a;a",
 				ColumnDefs = "a",
 			};
 
-			uiBuilder.Title = "Incorrect GQI Version";
+			uiBuilder.Title = "Unsupported GQI version";
```

Comment:
Does it make sense to say "GQI" version here when the description only mentions the web app version (and the GQI DxM).

Reply:
The user doesn't really need to know the technical background, so "Unsupported version" is probably more than enough. The text in the popup will tell them exactly what needs to be done.